cmd/protoc-gen-goclay/genhandler/handler.go

```diff
@@ -194,10 +194,28 @@ func (g *Generator) generateImplServiceMethod(file *descriptor.File, svc *descri
 
 		glog.V(1).Infof("Will emit %s", output)
 
-		return []*plugin.CodeGeneratorResponse_File{{
+		result := []*plugin.CodeGeneratorResponse_File{{
 			Name:    proto.String(output),
 			Content: proto.String(string(formatted)),
-		}}, nil
+		}}
+
+		testCode, err := g.getImplTemplate(file, svc, method, true)
```

Comment:
I think it'd be a good idea to add a flag that would disable testfile generation when it's not needed.

Reply:
WIP. :)

Reply:
Done. Please check it with [329f0a99bbb8cb02262d466e1308d4accd7ff9d8](https://github.com/utrack/clay/pull/82/commits/329f0a99bbb8cb02262d466e1308d4accd7ff9d8)